docs/en/Building-Alluxio-Master-Branch.md

```diff
@@ -1,69 +1,90 @@
 ---
 layout: global
-title: Building Alluxio Master Branch
-nickname: Building Master Branch
+title: Building Alluxio From Source Code
```

Comment:
should we rename the file now and update references?

Reply:
done